app/models/color_style.rb

```diff
@@ -0,0 +1,7 @@
+class ColorStyle < ApplicationRecord
+  belongs_to :team
+
+  validates :name, presence: true
+  validates :color_value, presence: true
+  validates :team_id, presence: true
+end
```

Comment:
Ensure that the 'color_value' field is validated for proper color format, not just presence, to maintain data integrity.